test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -46,7 +46,7 @@ package require Tcl 8.0
 proc ::AlgorandGoal::Abort { ERROR } {
     puts "Aborting with Error: $ERROR"
 
-    if { "$::GLOBAL_TEST_ALGO_DIR" != "" && "$::GLOBAL_TEST_ROOT_DIR" != "" } {
+    if { [info exists ::GLOBAL_TEST_ALGO_DIR] && [info exists ::GLOBAL_TEST_ROOT_DIR] } {
```

Comment:
this doesn't really related to this PR , right ?

Reply:
no. But when the test was failing, the abort script was generating more errors. I'll make this into a separate PR.